apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimTransformerV2.java

```diff
@@ -314,14 +318,19 @@ private static Money getTotal(PreAdjMcsClaim claimGroup) {
             Optional.ofNullable(detail.getIdrModThree()),
             Optional.ofNullable(detail.getIdrModFour()));
 
+    List<String> systemSuffix = List.of("one", "two", "three", "four");
+
     // OptionalGetWithoutIsPresent - IsPresent used in filter
     //noinspection OptionalGetWithoutIsPresent
     return IntStream.range(0, mods.size())
         .filter(i -> mods.get(i).isPresent())
         .mapToObj(
             index ->
                 new CodeableConcept(
-                    new Coding(BBCodingSystems.HCPCS, mods.get(index).get(), null)
+                    new Coding(
+                            BBCodingSystems.MCS.MOD_PREFIX + systemSuffix.get(index),
```

Comment:
Per a conversation with Bill Harty, lets use "https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets" as the system for MCS HCPCS modifier codes